### DISCUSSION TITLE
How to correctly pass sftp.args

### DISCUSSION THREAD

**Discussion by causalmask** [garethgeorge/backrest#145] [Q&A]
I'm having trouble correctly formulating the flags field to pass `-o "sftp.args=-i /path/to/private_key"` for example.  I can get this to work correctly calling restic from the command line within the backrest docker container, but I can't seem to get it to work when attempting to configure it via the backrest web UI.  

Is there some sort of special whitespace string handling or escape characters I need to be using?

I'm also not sure where to look for error logs.  There's a popup error message at the top of the web UI but all it says is that it can't authenticate, so I'm not sure if the flags I pass are ending up in the right place.

**Comment by arminus** [comment]
-o sftp.args="-i C:/Users/myself/.ssh/id_rsa"

**Comment by Pink-o** [comment]
Maybe you are having my same problem [209](https://github.com/garethgeorge/backrest/discussions/209) I was using a key with a passphrase, I ended up creating a new key without and it works also in the webUI
